src/pages/overview/pdf-services-api/howtos/webhook-notification.md

```diff
@@ -0,0 +1,123 @@
+  
+  "notifiers": [
+    {
+      "type": "CALLBACK",
+      "data": {
+        "url": "https://dummy.callback.org/",
+        "headers": {
+          "x-api-key": "dummykey",
+          "access-token": "dummytoken"
+        }
+      }
+    }
+  ]
+}
+```
+
+### Callback Payload on HTTPS URL
+The notification payload sent to CALLBACK URL is same as that of **GET STATUS** response along with jobID.
+
+#### In case Job completes with Success
+In case of successful job completion, the following **callback payload** will be sent to callback url:
+```json
+{
+  "jobID": "<JOB ID>",
+  "statusResponse": {
+    "status": "done",
+    "asset": {
+      "metadata": {
+        "type": "application/pdf",
+        "size": 318974
+      },
+      "downloadUri": "https://dcplatformstorageservice-dev-us-east-1.s3-accelerate.amazonaws.com/d2276912a0b243e696f124ff456be9b1_C9851F6663033A610A494128%40techacct.adobe.com/d515e6cc-4f4b-4b28-953a-7a08391df783?X-Amz-Security-Token=FwoGZXIvYXdzELz%2F%2F%2F%2F%2F%2F%2F%2F%2F%2FwEaDJJ5weJV7WO8%2FQF1HSLTAbfoqPU3TaJWlIXdHRqqFd30LHw3r6eWyLQMpESoer%2Bf%2BArqHQXr41qlrhXlocuiateFjV3wDRWxrATCXOM3Sd%2BzG4xOdPqWTlqSS4kdichFW6YxZGQWAX%2BYG4UkmsRMhqiNb0YJzibgraDjEwEdqcS%2B4jh5eVaS3FxkjbsP6IfqxUylwElwgAkplhhwv0bXel%2BoJOMcNKY0RKpjacmd5WO3heHycxTikl22H7nkN2lbgoKvD9%2Bch%2FXyaZJ16GRxUsLsJYkHMseSel3yBZa3dVNCWu4ouL%2BNqQYyLeQUtRlODiJdMXRktx%2FbQ56xDgJKRbxD%2BKTZlfH44GeCwh%2B9GJCmchbyW54NbA%3D%3D&X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Date=20231009T021610Z&X-Amz-SignedHeaders=host&X-Amz-Expires=3599&X-Amz-Credential=ASIAU5PA7W47EACOK46J%2F20231009%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Signature=44e6060ca6bb1e70ec30b150737b4ee196acb5842efc5dd7adf05e9d1ad84074",
+      "assetID": "urn:aaid:AS:UE1:786e4ea5-a720-4d0a-af6d-cb450719989c"
+    }
+  }
+}
+```
+
+#### In case Job completes with Failure
+In case of failed job completion, the following **callback payload** will be sent to callback url:
+```json
+{
+  "jobID": "<JOB ID>",
+  "statusResponse": {
+    "status": "failed",
+    "error": {
+      "code": "INVALID_PROVIDER_NAME",
+      "message": "Provider with name csctest not supported.; transactionId=c0uyOOfEtj04aWSeLiesgA6TQ4WoLc2b",
+      "status": 400
+    }
+  }
+}
+```
+
+## Expected Response from the Client
+The client is expected to return the following response payload with an HTTP status code of 200 (OK):
+```json
+{
+  "ack": "done"
+}
+```
+<InlineAlert slots="text"/>
+<div>
+If the response code is not 200 (OK) or the expected payload is not received, it will be considered an error response. In the event of 50 error responses within a 10-minute period, webhook notification support will be temporarily blocked for that client for the next 20 minutes.
+</div>
+
+## Limitations
+1. Webhook notification support is currently available only in internal storage. It is currently not supported for external storage.
+2. This support is not available for PDFProperties (irrespective of whether it is internal or external storage).
```

Comment:
It is not available for PDFProperties (link the operation technical documentation link) operation.